pennylane/measurements/mutual_info.py

```diff
@@ -76,4 +79,17 @@ def circuit_mutual(x):
 
     wires0 = qml.wires.Wires(wires0)
     wires1 = qml.wires.Wires(wires1)
-    return MeasurementProcess(MutualInfo, wires=[wires0, wires1], log_base=log_base)
+    return _MutualInfo(MutualInfo, wires=[wires0, wires1], log_base=log_base)
+
+
+class _MutualInfo(StateMeasurement):
+    """Measurement process that returns the mutual information."""
+
+    def process_state(self, state: Sequence[complex], wires: Wires):
+        return qml.math.mutual_info(
+            state,
+            indices0=[self.wires[0]],
+            indices1=[self.wires[1]],
```

Comment:
Since there are two sets of wires passed to this measurement, I think it should be `indices0=self.raw_wires[0].tolist()` and `indices1=self.raw_wires[1].tolist()`. Before you make this change, can you test if the original returns the correct result with multiple measured wires?

Reply:
Nice catch! Changed to `list(self._wires[0])`